### DISCUSSION TITLE
homepage lxc error

### DISCUSSION THREAD

**Discussion by PirateJvck** [tteck/Proxmox#3947] [Help Needed]
__  __                                          
   / / / /___  ____ ___  ___  ____  ____ _____ ____ 
  / /_/ / __ \/ __ `__ \/ _ \/ __ \/ __ `/ __ `/ _ \
 / __  / /_/ / / / / / /  __/ /_/ / /_/ / /_/ /  __/
/_/ /_/\____/_/ /_/ /_/\___/ .___/\__,_/\__, /\___/ 
                          /_/          /____/       
 -Browserslist: caniuse-lite is outdated. Please run:
  npx update-browserslist-db@latest
  Why you should do it regularly: https://github.com/browserslist/update-db#readme
Latest version:     1.0.30001669
Updating caniuse-lite version
$ pnpm up caniuse-lite
 |caniuse-lite has been successfully updated

No target browser changes
 |Lockfile is up to date, resolution step is skipped
 -Already up to date
 |Done in 1.6s
 -
> homepage@0.9.6 build /opt/homepage
> next build

Failed to compile.

./src/widgets/customapi/component.jsx
38:10  Error: Unnecessary 'else' after 'return'.  no-else-return

info  - Need to disable some ESLint rules? Learn more here: https://nextjs.org/docs/basic-features/eslint#disabling-rules
 - ELIFECYCLE  Command failed with exit code 1..

[ERROR] in line 78: exit code 0: while executing command pnpm build

root@homepage:/#

**Comment by tteck** [comment]
🏁 The script runs without any problems from this end.

```
    __  __                                          
   / / / /___  ____ ___  ___  ____  ____ _____ ____ 
  / /_/ / __ \/ __ `__ \/ _ \/ __ \/ __ `/ __ `/ _ \
 / __  / /_/ / / / / / /  __/ /_/ / /_/ / /_/ /  __/
/_/ /_/\____/_/ /_/ /_/\___/ .___/\__,_/\__, /\___/ 
                          /_/          /____/       
Using Advanced Settings
Using Distribution: debian
Using debian Version: 12
Using Container Type: 1
Using Root Password: Automatic Login
Container ID: 111
Using Hostname: homepage
Using Disk Size: 3
Allocated Cores: 2
Allocated RAM: 1024
Using Bridge: vmbr0
Using IP Address: dhcp
Using Gateway IP Address: Default
Using APT-Cacher IP Address: Default
Disable IPv6: yes
Using Interface MTU Size: Default
Using DNS Search Domain: Host
Using DNS Server IP Address: Host
Using MAC Address: AE:1A:60:07:B7:59
Using Vlan: Default
Enable Root SSH Access: no
Enable Verbose Mode: no
Creating a Homepage LXC using the above advanced settings
 ✓ Using local for Template Storage.
 ✓ Using thin-pool for Container Storage.
 ✓ Updated LXC Template List
 ✓ LXC Container 111 was successfully created.
 ✓ Started LXC Container
 ✓ Set up Container OS
 ✓ Network Connected: 192.168.86.250 
 ✓ IPv4 Internet Connected
 ✗ IPv6 Internet Not Connected
 ✓ DNS Resolved github.com to 140.82.113.4
 ✓ Updated Container OS
 ✓ Installed Dependencies
 ✓ Set up Node.js Repository
 ✓ Installed Node.js
 ✓ Installed Homepage v0.9.11
 ✓ Created Service
 ✓ Customized Container
 ✓ Cleaned
 ✓ Completed Successfully!

Homepage Setup should be reachable by going to the following URL.
         http://192.168.86.250:3000 
```

**Comment by giannis371** [comment]
Same here

Failed to compile.

./src/widgets/customapi/component.jsx
38:10  Error: Unnecessary 'else' after 'return'.  no-else-return

info  - Need to disable some ESLint rules? Learn more here: https://nextjs.org/docs/basic-features/eslint#disabling-rules
info  - Linting and checking validity of types . ELIFECYCLE  Command failed with exit code 1.

[ERROR] in line 78: exit code 0: while executing command pnpm build

**Reply by PirateJvck** [reply]
I’m not sure but when I checked my container the drive was full I just haven’t had a chance to fix it yet

**Reply by giannis371** [reply]
I restarted the container and it worked
